### ISSUE TITLE
Could you tell me how to run it?

### ISSUE THREAD

**Issue by qidaneix** [cocagne/paxos#5]
what should I do if I want to running it as the beginning of the Paxos  algorithm？

**Comment by cocagne** [comment]
I'm not sure what you're asking. A bit more context surrounding the
question would be helpful.

Tom

**Comment by qidaneix** [comment]
I am so sorry that my English is not very well(I'm Chinese).
I want to learn Paxos algorithm. I have read your code. But I don't know how to run it as a Paxos algorithm.
Just like you give me a cellphone, but I don't know how to use it to make a phone call.
Thank you to tell me

Jin

**Comment by cocagne** [comment]
First, read this: https://understandingpaxos.wordpress.com/

That will explain the algorithm in detail (sorry, no Chinese translation).
Next, use this implementation instead:
https://github.com/cocagne/python-composable-paxos

That implementation is cleaner and easier to use.

Good luck!

Tom

**Comment by qidaneix** [comment]
@cocagne Hi Tom!

I have read the implementation:
[https://github.com/cocagne/python-composable-paxos](url)
you write these days. 
And understand each role in Paxos algorithm somewhat.

But I still want some example about a Paxos algorithm running, which can get the runing  results.

Could you give me some suggestion? Thank you.

Best Wishes!

Jin

**Comment by cocagne** [comment]
Jin,

I think what you're looking for is available here:
https://github.com/cocagne/multi-paxos-example

I wrote this specifically for people looking for a minimal but fully
functional Paxos implementation that can be used to aid in understanding
the algorithm. The code is relatively short and highly compartmentalized
and there is a good amount of documentation to aid the learning process.
However, a realistic expectation is that it will likely take several days
of effort to fully comprehend. I've tried to make it as easy as possible
but it's still a complex problem.

Good luck,

Tom
